File: Assignment_6/Q5.c

///////////////////////////////////////////////////////
//
//  File name :     assignment6-2.c
// Descreption : write  program which accept number from user and display its table in reverse
//  Author :        Vaishanvi D Shingare
//  Date :          28/04/2025

//input : 2
// output : 20  18  16  14  12  10  8  6   4   2 
//input : 5
//output : 50  45  40  35  30  25  20  15  10  5

//input : -5
//output : 50  45  40  35  30  25  20  15  10  5

///////////////////////////////////////////////////////



#include<stdio.h>

void TableRev(int iNo)
{
    if (iNo < 0)
    {
        iNo = -iNo;
    }
    

    for(int i = 10 ; i >= 1  ; i--)
    {
        printf("%d \t",i * iNo);
    }
   
}

int main()
{
    int iValue = 0; 

    printf("Enter number :");
    scanf("%d",&iValue);

    TableRev(iValue);
    
    return 0;
}

File: Assignment_49/Q5.java
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     Q5.java
//  Description : write java program which accept string from user and display below pattern.
//  Author :        Vaishanvi D Shingare
//  Date :          2/6/2025
/*
// INPUT  : Hello
//  OUTPUT :
            H  *  *  *  *
            H  e  *  *  *
            H  e  l  *  *
            H  e  l  l  *
            H  e  l  l  o
            H  e  l  l  *
            H  e  l  *  *
            H  e  l  *  *
            H  e  *  *  *
            H  *  *  *  *
*/         
///////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Q5
{
    public void Pattern(String str) 
    {
        int ino = str.length();
        int i = 0;
        int j = 0;

        for ( i = 0 ; i < ino ; i++) 
        {
            for ( j = 0; j < ino ; j++) 
            {
                if(i <= j)
                {
                     System.out.print("* ");

                }
                else
                {
                     System.out.print(str.charAt(j) + " ");

                }
            }
            System.out.println();
        }
        for ( i = 0 ; i < ino; i++) 
        {
            for ( j = 0; j < ino; j++) 
            {
                if(j < (ino - i))
                {
                    System.out.print(str.charAt(j) + " ");
                }
                else
                {
                    System.out.print("* ");

                }
            }
            System.out.println();
        }
    }
    
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.print("Enter a string: ");
        String input = sobj.nextLine();

        Q5 obj = new Q5();
        obj.Pattern(input);
    }
}